src/test/java/com/textadventure/MainTest.java

```diff
@@ -8,4 +14,25 @@
     public void testMain() {
         Assert.assertEquals(true, true);
     }
+    @Test
+    public void testOutputToken() {
+    	String inputTest = " W S North run walk D ";
+    	Lexer lexer = new Lexer();
```

Comment:
If your lex method is static, why not call it in a static way? Lexer.lex will work too, no need for an instance of a class.